### ISSUE TITLE
Documents not getting delivered.

### ISSUE THREAD

**Issue by local-boy** [cdhigh/KindleEar#422]
I have deployed the kindleear app using glcoud successfully. Everything is working fine on the app but nothing is getting delivered to my device even after trying everything and several times. please help in this regard. Following is the log when I click to deliver. Nothing is shown on the kindlear log. 

https://console.cloud.google.com/logs/viewer?src=ac&project=anasarifkindle&duration=PT1H&key1=default&minLogLevel=0&expandAll=false&resource=gae_app&logName=projects%2Fanasarifkindle%2Flogs%2Fappengine.googleapis.com%252Frequest_log&timestamp=2017-08-05T12%3A22%3A33.229770000Z&advancedFilter=resource.type%3D%22gae_app%22%0Aresource.labels.version_id%3D%2220170803t104102%22%0Aresource.labels.module_id%3D%22default%22%0Aresource.labels.zone%3D%22us12%22%0Aresource.labels.project_id%3D%22anasarifkindle%22%0Atimestamp%3D%222017-08-05T12%3A19%3A59.418836000Z%22%0AinsertId%3D%225985b7ef000a2fd3bf6f163d%22

https://console.cloud.google.com/logs/viewer?src=ac&project=anasarifkindle&duration=PT1H&key1=default&minLogLevel=0&expandAll=false&resource=gae_app&logName=projects%2Fanasarifkindle%2Flogs%2Fappengine.googleapis.com%252Frequest_log&timestamp=2017-08-05T12%3A22%3A33.229770000Z&advancedFilter=resource.type%3D%22gae_app%22%0Aresource.labels.zone%3D%22us12%22%0Aresource.labels.project_id%3D%22anasarifkindle%22%0Aresource.labels.version_id%3D%2220170803t104102%22%0Aresource.labels.module_id%3D%22default%22%0Atimestamp%3D%222017-08-05T12%3A19%3A59.686390000Z%22%0AinsertId%3D%225985b7f0000a2ffeb1d80dbf%22

https://console.cloud.google.com/logs/viewer?src=ac&project=anasarifkindle&duration=PT1H&key1=default&minLogLevel=0&expandAll=false&resource=gae_app&logName=projects%2Fanasarifkindle%2Flogs%2Fappengine.googleapis.com%252Frequest_log&timestamp=2017-08-05T12%3A22%3A33.229770000Z&advancedFilter=resource.type%3D%22gae_app%22%0Aresource.labels.module_id%3D%22default%22%0Aresource.labels.zone%3D%22us12%22%0Aresource.labels.project_id%3D%22anasarifkindle%22%0Aresource.labels.version_id%3D%2220170803t104102%22%0Atimestamp%3D%222017-08-05T12%3A20%3A59.768032000Z%22%0AinsertId%3D%225985b82b000d245e7cb5469d%22

**Comment by cdhigh** [comment]
You can confirm Module 'worker' is actived or not, if you deployed KindleEar using gclound, maybe you forgot to deploy 'worker'.

**Comment by local-boy** [comment]
I don't know much of the programming and nothing of python and gae. Can you please guide me how to check whether worker is active or not. When I deploy module-worker.yaml using gcloud then web application shows 'not found'. After deploying app.yaml again, app is accessible but documents are still not being sent. I get this error when i try to deploy both simultaneously.

C:\Users\iAnas\Downloads\KindleEar>gcloud app deploy app.yaml module-worker.yaml
ERROR: (gcloud.app.deploy) [C:\Users\iAnas\Downloads\KindleEar\app.yaml] and [C:\Users\iAnas\Downloads\KindleEar\module-worker.yaml] are both defining the service id [default]. All <service>.yaml files must have unique service ids.

**Comment by local-boy** [comment]
Sorry for the trouble i gave you. I have found the mistake now. I was actually deleting the module: worker from the module-worker.yaml and it is being deployed then as default service. I changed module: worker to service:worker and redeployed and now everything is working fine. Thanx a lot for this wonderful application. God bless you!!

Additionally I have one request that you update the deployment process now since old method no more works as there is no google_appengine folder in gcloud now. It will be helpful to newbies like me.
